<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>runWindow</class>
 <widget class="QMainWindow" name="runWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="driveinLabel">
      <property name="text">
       <string>Drive in Queue</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="driveinTable"/>
    </item>
    <item>
     <widget class="QLabel" name="insideLabel">
      <property name="text">
       <string>Inside Queue</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="insideTable"/>
    </item>
    <item>
     <widget class="QLabel" name="resultsLabel">
      <property name="text">
       <string>Results</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="resultsTable"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
